content/composer/overview.md

```diff
@@ -9,39 +9,78 @@ weight: 200
 
 ## Logging In
-The first time that you use Cloudify Composer, you must supply a username and a password. Use the following defaults:<br />
-**Username:** `composer` <br />
-**Password:** `composer`
 
-When you have logged in to Cloudify Composer, the Topology page for the selected blueprint is displayed.
+To use the Cloudify Composer component, you log into Cloudify Manager. If you are already in Cloudify Manager, click the ** Cloudify Composer* button in the **Local blueprints** screen.
+
+When you have logged in to Cloudify Composer, the default Topology page is displayed, with an empty blueprint for your use.
 
 ![Blueprints List]({{< img "composer/composer_interface.png" >}})
 
-The left side of the Cloudify Composer Topology tab displays a Stencils catalog that contains node types that are used as the building blocks of the topology. The main pane on the tab is a canvas on to which you drag and drop nodes and define the relationships between them. 
+The left side of the Cloudify Composer Topology tab displays a Stencils pane that contains node types that can be used as the building blocks of the topology. The main pane on the tab is a canvas on to which you can drag and drop nodes and define the relationships between them. 
+
+You can click the Star icon in any node type in the Stencils pane to mark it as a favorite and move it to the Favorites section of the Stencils pane.
 
 
 ## Workflow
 This section describes the main functions related to creating a Blueprint. Typically, when you are using Cloudify Composer, your workflow will follow a similar order to that described in this section. 
 
-Additional functions, such as importing Stencils and Blueprints, and so on, are described after the primary functions related to creating a Blueprint.
+### Importing Stencils###
+
+Before you start to design your blueprint, you will typically want to add the stencils that contain the basic node types that you need to work with, and the operations that they expose. For more information about node types, [click here]({{< relref "blueprints/spec-node-types.md" >}}).
+
+Cloudify Composer supports two methods for adding node types to your Stencils catalog.
+
+#### Import Method 1
+
+You can import a `.yaml` file that contains definitions of multiple node types. These files are referred to as _stencils_.
+  
+**Import a Stencil**
+
+* Click the **Imports** tab and select one of the following options:
+
+  * Select a default Cloudify plugin in the catalog and click **Add**.
+  * Click **Add new import** and specify a URL or local file that you want to add, and click **Save**.
+
+After you have imported a stencil, it appears in the Imports list and you can see all the node types that were added, in their relevant node type group. The node types can be added to your blueprint package.
```

Comment:
The last sentence (The node types can be added to your blueprint package) should be removed. I didn't notice it earlier.